src/backend/utils/appConfig.ts

```diff
@@ -112,6 +112,46 @@ const AppConfig = z.object({
 
 export type AppConfig = z.infer<typeof AppConfig>;
 
+const defaultConfig: AppConfig = {
+  'port': 3000,
+  'host': 'localhost',
+  'uploads': {
+    'driver': 'local',
+    'local': {
+      'path': './uploads',
+    },
+  },
+  'frontend': {
+    'title': 'CodeX Docs',
+    'description': 'A block-styled editor with clean JSON output',
```

Comment:
why the default config is hardcoded inside the app?

Reply:
because it's just default values if user will not pass some values 🤷‍♂️
I don't see any sense to put it in file
the user can specify only those config fields that he really needs

Reply:
but there are our values from Editor.js. Let's change them for some more abstract, and remove optional values like `misprintsChatId`